impl/src/license.rs

```diff
@@ -40,30 +45,143 @@ impl BazelLicenseType {
   }
 }
 
-/** Breaks apart a cargo license string and yields the available license types. */
-pub fn get_available_licenses(cargo_license_str: &str) -> Vec<(String, BazelLicenseType)> {
-  let mut available_licenses = Vec::new();
-  for license_name in cargo_license_str.split('/') {
-    if license_name.is_empty() {
-      continue;
+/**
+ * A data structure for calculating a crate's license restrictions
+ */
+#[derive(Debug)]
+pub struct BazelSpdxLicense {
```

Comment:
Probably doesn't need to be pub.

Reply:
`get_available_licenses` currently returns this new type. Should I just make it return `LicenseData`?

Reply:
Oh, nevermind. I think I was under the impression that this was redundant with some other "processing finished" type, but I think you're actually returning instances of this type from the license functino.

Reply:
Actually yes, if that's possible then that is probably better.

I have some preference for having a "declarative", "simple" type for the values returned from the module, but I don't feel strongly about it, and I think "convenience" is a good enough argument for whatever you choose to do.

Reply:
Okay, newest commit has this change